PackagingProjects/CSharpPluginPayload/CSharpPluginPayload.csproj

```diff
@@ -29,12 +29,14 @@
     <Optimize>true</Optimize>
     <OutputPath>bin\Release\</OutputPath>
     <DefineConstants>TRACE</DefineConstants>
+    <NoWarn>CS2008</NoWarn>
```

Comment:
Miscellaneous fix: suppress the compiler warning that the packing project does not have any .cs files.